dpnp/dpnp_iface_indexing.py

```diff
@@ -227,7 +226,8 @@ def _take_1d_index(x, inds, axis, q, usm_type, out=None):
 
 def compress(condition, a, axis=None, out=None):
     """
-    Return selected slices of an array along given axis.
+    A copy of `a` without the slices along `axis` for which `condition` is
```

Comment:
Should it be moved to `Returns`? It seems in numpy that exact description is used to describe the returned result.

Reply:
CuPy is also pretty particular with the language in this way, I'll apply this change